src/app/content/components/Page/PageContent.tsx

```diff
@@ -89,7 +89,8 @@ export default styled(MainContent)`
         }
       }
 
-      &.first.text.has-note:after {
+      &.first.text.has-note [data-for-screenreaders]:first-child {
```

Comment:
why did you change this to from `:after`

Reply:
this was an unreverted change for my attempt to fix [this bug](https://drive.google.com/file/d/1ZsrkH0mDZbRK5PD9dU2YppcBG4st6KEz/view).